src/components/App/App.tsx

```diff
@@ -24,11 +55,42 @@ const App = (): JSX.Element => {
             <Route exact path="/">
               <Form updateUserSentiment={ updateUserSentiment } />
             </Route>
-            <Route path="/feed/">
-              <Feed userSentiment={ userSentiment } />
-            </Route>
+            <Route
+              exact path="/feed/"
```

Comment:
We can make this more future-proof by leaving off the trailing `/`

```suggestion
              exact path="/feed"
```